examples/webgl_modifier_subdivision.html

```diff
@@ -205,51 +211,40 @@
 
 				// Cloning original geometry for debuging
 
-				smooth = geometry.clone();
+				//smooth = geometry.clone(); // removed because the BufferSubdivisionModifier doesn't modify the original object.
 
-				// mergeVertices(); is run in case of duplicated vertices
-				smooth.mergeVertices();
-				smooth.computeFaceNormals();
-				smooth.computeVertexNormals();
+				// mergeVertices(); is run in the subdivision modifier 
```

Comment:
also, the normals are computed post subdivision by the subdiv mod, so no need to run that now.